### PR TITLE
Add commands 'stacks' and 'stacks-set' to list and manipulate apps runtime stack

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: cmd/stacks.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	"github.com/Scalingo/cli/stacks"
+	"github.com/urfave/cli"
+)
+
+var (
+	stacksListCommand = cli.Command{
+		Name:     "stacks",
+		Category: "Runtime Stacks",
+		Usage:    "List the available runtime stacks",
+		Description: ` List all the available rutime stacks for your apps:
```
Comment (EtienneM) [inline cmd/stacks.go]:
```suggestion
		Description: ` List all the available runtime stacks for your apps:
```

File: stacks/set.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+func Set(app string, stack string) error {
+	c := config.ScalingoClient()
+	stacks, err := c.StacksList()
+	if err != nil {
+		return errgo.Notef(err, "fail to list available stacks")
+	}
+
+	var stackToSet scalingo.Stack
+	for _, apistack := range stacks {
```
Comment (EtienneM) [inline stacks/set.go]: Is it mandatory to do all this? `AppsSetStack` should already test that the stack exists before it sets it, shouldn't it?

Comment (Soulou) [reply stacks/set.go]: It's to get the ID, because we can't set a stack by its name.

**Comment by Soulou** [comment]
Rebased

File: stacks/set.go (hunk 2 above)
Comment (EtienneM) [reply stacks/set.go]: OK but don't we want to update the `AppsSetStack` method to accept both ID and name, like for some other API endpoint?

Comment (Soulou) [reply stacks/set.go]: Hmmm are we doing this? For which endpoint, can you tell me?

Comment (EtienneM) [reply stacks/set.go]: You're right, we don't do that. I supposed we do that for the apps endpoint in which we can use both the ID or the name but apparently no.

**Pull request merged**